gameplay/game.go

```diff
@@ -127,16 +127,19 @@ func (game *Game) Start() {
 	game.PlayerOne.SendPacket(networking.Packet{PacketID: networking.OpponentInitiationInformation, Content: selfContent})
 
 	// Send players packets with their starting hands
-	rand.Seed(time.Now().UnixNano())
-	hand := []Card{Cards[1], Cards[1], Cards[2], Cards[2]}
-
-	packetContent := map[string]interface{}{
-		"hand": hand,
+	for i := 0; i < 30; i++ {
+		rand.Seed(time.Now().UnixNano())
```

Comment:
It's probably not necessary to reseed the random library every time it goes through the loop as this uses extra processing and wont add to the randomness. It is instead easier to see the random library before the for loop.